dev-lang/elm-compiler-bin/elm-compiler-bin-0.19.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The 'elm' command line interface (binary)"
+HOMEPAGE="http://elm-lang.org"
+SRC_URI="https://github.com/elm/compiler/releases/download/${PV}/binary-for-linux-64-bit.gz -> ${PV}.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="+symlink"
+
+RDEPEND=" >=dev-haskell/ansi-terminal-0.8:=
```

Comment:
If this package just unpacks the bianry these RDEPENDs are probably not needed.

Reply:
I figured it would need them at runtime, like a dynamic library or something, no?

Reply:
Aren't you installing prebuilt binary? That's very unlikely anyone relies on haskell shared libraries installed by the system.

`lddtree` can show actual dynamic libraries used.